### PR TITLE
Release for brahma 0.0.3

### PR DESCRIPTION
**Pull request by hariharan-devarajan**

### CODE REVIEW

File: var/spack/repos/builtin/packages/brahma/package.py [hunk 1]
```diff
@@ -17,8 +17,9 @@ class Brahma(CMakePackage):
 
     version("develop", branch="dev")
     version("master", branch="master")
-    version("0.0.2", tag="v0.0.2", commit="4a36d5c08787d41c939fa1b987344b69d9ef97a6")
     version("0.0.1", tag="v0.0.1", commit="15156036f14e36511dfc3f3751dc953540526a2b")
+    version("0.0.2", tag="v0.0.2", commit="4a36d5c08787d41c939fa1b987344b69d9ef97a6")
+    version("0.0.3", tag="v0.0.3", commit="fd201c653e8fa00d4ba6197a56a513f740e3014e")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/brahma/package.py]: The convention is to list newer version directives first.  You can see an example in the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#versions-and-fetching.

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed the new commits exist.  Please reorder the version directives.

File: var/spack/repos/builtin/packages/brahma/package.py (hunk 1 above)
Comment (hariharan-devarajan) [reply var/spack/repos/builtin/packages/brahma/package.py]: DONE

**Review by tldahlgren — APPROVED** [review]
Thanks.

**Pull request merged**